def failures(self):
    return self.__failures
